VultisigApp/VultisigApp/View Models/KeysignDiscoveryViewModel.swift

```diff
@@ -253,4 +267,11 @@ class KeysignDiscoveryViewModel: ObservableObject {
         }
     }
     
+    private var keysignType: KeyType {
+        if let keysignPayload {
+            return keysignPayload.coin.chain.signingKeyType
+        } else {
+            return .ECDSA
```

Comment:
Keep in mind , we should allow user to choose what keys they want to sign the tx with , but for now , I guess it will be fine